packages/app/src/D2Shim.js

```diff
@@ -6,7 +6,6 @@ import { useState } from 'react'
 import { init as d2Init, config as d2Config, getUserSettings } from 'd2'
```

Comment:
Not sure, but this could be the right time to remove this and use the `app-runtime-adapter-d2`? There are some things in this file that are not in that lib though: `history`, and `extractUserSettings` so those would have to be done elsewhere

Reply:
I'm going to look into it.

Reply:
Those should be able to be done in the callbacks in the lib, I believe

Reply:
I started removing this shim in favour of the one from the adapter.
I'll push the changes soon.

Reply:
Will add these changes in a separate PR.
This one is growing too big.